interactionCommands.json

```diff
@@ -431,6 +431,10 @@
 							{
 								"name": "cs-CZ",
 								"value": "cs-CZ"
+							},
+							{
+								"name": "it-IT",
```

Comment:
Perhaps we could do the names for these as each language's proper name, in that language?

Reply:
Def something we do in the future